### PR TITLE
chore: [sc-18945] Review all copyright messages in source code and make sure they are greymatter.io, not Decipher.

### PR DESCRIPTION
**Pull request by abhay-greymatter**
Story details: https://app.shortcut.com/grey-matter/story/18945

### CODE REVIEW

**Review by ChocoTaco12 — APPROVED** [review]

File: LICENSE [hunk 1]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2022 Decipher Technology Studios, LLC; DBA Grey Matter software
+   Copyright 2022 greymatter.io; DBA Grey Matter software
```
Comment (tilleryd) [inline LICENSE:189]:
```suggestion
   Copyright 2022 greymatter.io Inc; DBA Grey Matter software
```

File: api/header.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright Decipher Technology Studios, LLC; DBA Grey Matter software 2022.
+Copyright greymatter.io; DBA Grey Matter software 2022.
```
Comment (tilleryd) [inline api/header.txt:2]:
```suggestion
Copyright greymatter.io Inc; DBA Grey Matter software 2022.
```

**Review by tilleryd — CHANGES_REQUESTED** [review]
Just one comment on Inc versus LLC.

**Review by tilleryd — APPROVED** [review]

File: api/v1alpha1/zz_generated.deepcopy.go [hunk 3]
```diff
@@ -2,7 +2,7 @@
 // +build !ignore_autogenerated
 
 /*
-Copyright Decipher Technology Studios, LLC; DBA Grey Matter software 2022.
+Copyright greymatter.io; DBA Grey Matter software 2022.
```
Comment (abhay-greymatter) [inline api/v1alpha1/zz_generated.deepcopy.go:5]: @ChocoTaco12 This one (and others) should have Inc. because we replaced LLC

**Pull request merged**